data_models/bioinformatics_mill_models/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.utils import timezone
+
+
+class TimeTrackedModel(models.Model):
+    created_at = models.DateTimeField(editable=False)
+    updated_at = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
```

Comment:
I found a comment online mentioning that those fields are considered evil and should be avoided. The explanation linked [here](http://stackoverflow.com/questions/1737017/django-auto-now-and-auto-now-add/1737078#1737078) is where I got this method of doing this.